packs/feats/burn-it.json

```diff
@@ -48,14 +48,6 @@
                 "damageCategory": "persistent",
                 "damageType": "fire",
                 "key": "FlatModifier",
-                "predicate": [
-                    {
-                        "or": [
-                            "item:trait:alchemical",
-                            "item:type:spell"
-                        ]
-                    }
-                ],
```

Comment:
`You also gain a +1 status bonus to any persistent fire damage you deal.` We'd need to keep a predicate for the damage type and category.

Reply:
It'll work without

Reply:
(note there wasn't one to begin with)